include/evmc/evmc.h

```diff
@@ -163,10 +163,11 @@ typedef void (*evmc_get_tx_context_fn)(struct evmc_tx_context* result,
  *  @param      context  The pointer to the Host execution context.
  *  @param      number   The block number. Must be a value between
  *                       (and including) 0 and 255.
+ *  @return              1 if exists, 0 otherwise.
  */
-typedef void (*evmc_get_block_hash_fn)(struct evmc_uint256be* result,
-                                       struct evmc_context* context,
-                                       int64_t number);
+typedef int (*evmc_get_block_hash_fn)(struct evmc_uint256be* result,
```

Comment:
Should it return `evmc_status_code` (https://github.com/ethereum/evmc/blob/master/include/evmc/evmc.h#L191)?

Reply:
I don't think so, but we could consider having another enum for more regular cases.

Success/failure/invalid args.

Reply:
@chfast do we want an enum instead?

Reply:
I don't know yet. I have the review all similar cases and create the superset of possible error codes. If we end up with up to 4 items we can use it in all cases.

Reply:
We can merge it with `int`, but you have to improve documentation of `@result` and `@param number`.

Reply:
Should we wait for #116 first?

Reply:
The #116 is ready, just needs review.